### PR TITLE
zh-translation:/intro/arch_overview/observability/observability.rst

### PR DESCRIPTION
**Pull request by luohua13**
For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
[Optional Runtime guard:]
[Optional Fixes #Issue] https://github.com/cloudnativeto/envoy/issues/85
[Optional Deprecated:]

### CODE REVIEW

File: docs/root/intro/arch_overview/observability/observability.rst [hunk 1]
```diff
@@ -1,9 +1,9 @@
-Observability
+可观察性
 =============
 
 .. toctree::
   :maxdepth: 2
 
-  statistics
```
Comment (majinghe) [inline docs/root/intro/arch_overview/observability/observability.rst]: 这几个单词可以不用翻译，这个是rst的链接文件。这个文件 只翻译 最上面的 可观测性即可。下面的翻译过来之后文档编译会失败。

**Comment by majinghe** [comment]
@luohua13  添加了comment，处理一下。

File: docs/root/intro/arch_overview/observability/observability.rst (hunk 1 above)
Comment (luohua13) [reply docs/root/intro/arch_overview/observability/observability.rst]: Got it

**Pull request merged**